gcp_variant_transforms/beam_io/vcfio.py

```diff
@@ -53,6 +53,16 @@
 class _ToVcfRecordCoder(coders.Coder):
   """Coder for encoding :class:`Variant` objects as VCF text lines."""
 
+  def __init__(self, use_1_based_coordinate):
+    # type: (bool) -> None
+    """Initialize _ToVcfRecordCoder PTransform.
+
+    Args:
+      use_1_based_coordinate: specify whether the coordinates should be stored
+        in BQ using 1 based inclusive (default) or 0 based exclusive coordinate.
+    """
+    self.use_1_based_coordinate = use_1_based_coordinate
+
   def encode(self, variant):
     # type: (Variant) -> str
     """Converts a :class:`Variant` object back to a VCF line."""
```

Comment:
Hmm, I guess you didn't like my boolean arithmetic trick :).

We shouldn't add _start_1_based as an attribute to the class, because it would change with every invocation of the encode() method.

For the encode part, I think there is no need to create sample_1_based, since we can just inline the logic.

For the END tag part, I instead create start_0_based, because it makes length check easier.